packages/components/src/components/SquiggleViewer/utils.ts

```diff
@@ -126,7 +126,8 @@ export function useGetSubvalueByPath() {
 }
 
 export function getValueComment(value: SqValueWithContext): string | undefined {
-  return value.context.docstring() || value.tags.description();
+  const _value = value.context.docstring() || value.tags.description();
+  return _value && _value.length > 0 ? _value : undefined;
```

Comment:
Any thoughts on other functions to use instead of ``if (myString)``? I don't like relying on knowing which kinds of values are falsy in which situations. I could imagine using lodash methods or something, to be extra careful here.

Reply:
The only falsy string is `""`, e.g. `"0"` is true, so there should be no problem if you use TS (which will prevent you from multiplying a string and a number, or doing something else wat-like). It works the same as in Python and other sane languages.

Most of true/false insanity in JS is related to `==` operator (which is why we should disallow it in eslint). For reference: https://dorey.github.io/JavaScript-Equality-Table/.

Because of this, I think lodash falsy check here is excessive; it's useful when you're not sure about the variable type, e.g. if it's `string | null | undefined`, and you need to do multiple checks. But if you always pass strings then `myString === ""` is the only thing you should care about.

(there's a minor corner case where if the string comes from a form, you might want to treat `/\s*/` as falsy, in case someone accidentally types in a few spaces; but that should be normalized on the backend)